.vib/kong-ingress-controller/goss/kong-ingress-controller.yaml

```diff
@@ -0,0 +1,4 @@
+command:
+  check-app-binary:
```

Comment:
This is okay because it seems that `king-ingress-controller version` does not work properly. Bear in mind that you should create an internal task to try to address this issue:

```
$ ./kong-ingress-controller version
{"release":"NOT_SET","repo":"NOT_SET","commit":"NOT_SET"}
```